kafka-commons/src/main/java/uk/gov/homeoffice/digital/sas/kafka/consumer/KafkaConsumerService.java

```diff
@@ -21,16 +23,14 @@ protected KafkaConsumerService(SchemaValidator schemaValidator) {
     this.schemaValidator = schemaValidator;
   }
 
-  /**
-   * If payload is invalid consume returns null.
-   * The null value is up to the service to handle
-   */
-  public KafkaEventMessage<T> consume(String payload
+  public KafkaEventMessage<T> convertToKafkaEventMessage(String payload
   ) {
     if (schemaValidator.isSchemaValid(payload)) {
       log.info(String.format(KAFKA_CONSUMING_MESSAGE, payload));
-      return new Gson().fromJson(payload, new TypeToken<KafkaEventMessage<T>>() { }.getType());
+      return new Gson().fromJson(payload, new TypeToken<KafkaEventMessage<T>>() {
+      }.getType());
+    } else {
+      throw new ValidateException(String.format(KAFKA_SCHEMA_INVALID_VERSION, payload));
```

Comment:
From the package name this exception seems to be a security exception. Maybe use this one instead https://docs.spring.io/spring-kafka/api/org/springframework/kafka/support/serializer/DeserializationException.html 
or create your own checked exception